server/api/index.js

```diff
@@ -38,6 +38,16 @@ api.use(async (ctx, next) => {
       }
     }
 
+    if (message.match('Authorization error')) {
```

Comment:
This catches the error that's thrown by `authorize()` if the user does not have correct permissions.

I'm trying to match the pattern we had before here, which is return a `404` if attempting to read a document you don't have access to and a `401` if any other action.